docs/resources/rest_managed.md

```diff
@@ -81,10 +81,13 @@ resource "aci_rest_managed" "example_tenant_with_child" {
 
 * `annotation` - (string) Annotation for the class object that is being created.
   - Default: `orchestrator:terraform`
-* `content` (map) Map of key-value pairs those needed to be passed to the Model object as parameters. Make sure the key name matches the name with the object parameter in ACI.
+* `content` - (map) Map of key-value pairs those needed to be passed to the Model object as parameters. Make sure the key name matches the name with the object parameter in ACI.
```

Comment:
-> A map of key-value pairs needs to be passed to the Model object as parameters. Make sure the key name matches the name of the ACI object.

Reply:
fixed